bin/documentation.js

```diff
@@ -3,9 +3,11 @@
 /* eslint no-console: 0 */
 
 import yargs from 'yargs';
-import { hideBin } from 'yargs/helpers';
+import yargsHelpers from 'yargs/helpers';
```

Comment:
I didn't catch why is changed?

Reply:
This was brought up in https://github.com/yargs/yargs/issues/1789#issuecomment-717317765. The other option is to increment the yargs dependency itself which may inadvertently uncover other issues.

Reply:
@anthony-redFox I dropped the last commit after looking into it more. The underlying problem is the usage of the documentation binary in environments that are using mixed module resolution. Either way I separated it into a different branch so the original PR can proceed without problems. I'll open a new PR and explain the issue further there for the dropped commit.